timemachine/fe/free_energy.py

```diff
@@ -1204,7 +1195,7 @@ def replica_from_samples(traj: Trajectory) -> CoordsVelBox:
 
         fraction_accepted_by_pair_by_iter.append(fraction_accepted_by_pair)
 
-        if print_diagnostics_interval and iteration % print_diagnostics_interval == 0:
+        if print_diagnostics_interval and (current_frame + 1) % print_diagnostics_interval == 0:
```

Comment:
why +1 ?

Reply:
`current_frame` starts at 0, while `iteration` started at 1. This retains the same behavior (i.e. not printing the diagnostics after frame 0), but we could change it if preferable

Reply:
Should https://github.com/proteneer/timemachine/pull/1325/files/a682e19aa2943034c877764035414dd369b8a942#diff-6f4d3e6b5b0b6a38f1b9b5f8f51dcb56ec102ec03d06a7d53355bb67c4aee676R1187 be (current_frame+1) then? Not so concerned w/the diagnostics, just noticed the inconsistency between this line and the one above.

Reply:
I don't think it matters in that case, since we just need a different seed for each frame. I was on the fence about this, as adding 1 would maintain bitwise compatibility, but would also leave a weird arbitrary artifact in the code.

Reply:
I see, I'd prefer maintaining bitwise compatibility where possible.

Reply:
Didn't notice the seed, yeah should definitely maintain bitwise compatiblity

Reply:
OK, will add the +1, and a note that this can be cleaned up the next time we need to break binary compatibility

Reply:
Done in https://github.com/proteneer/timemachine/pull/1325/commits/002fc06efb4e926d42feacba180553ba202d06fd